### PR TITLE
modified runSim.sh for new docker container

### PR DESCRIPTION
**Pull request by puzzler7**

### CODE REVIEW

File: simulation/runSim.sh [hunk 1]
```diff
@@ -1,8 +1,8 @@
-COPPELIA_SIM_SCRIPT=~/dev/coppelia/coppeliaSim.sh
+COPPELIA_SIM_SCRIPT=~/docker-build/coppelia/coppeliaSim.sh
```
Comment (muthuArivoli) [inline simulation/runSim.sh:1]: Do you want this script in ~/dev/coppelia/coppeliaSim.sh? Should be an easy fix in the dockerfile if you want to move it.

File: simulation/runSim.sh [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 SERVER="comm_server_scene.ttt"
 echo "Starting CoppeliaSim"
 
-SERVER_LOCATION=$(find /home/duke/dev -name $SERVER | head -1)
+SERVER_LOCATION=$(find /root -name $SERVER | head -1)
```
Comment (muthuArivoli) [inline simulation/runSim.sh:5]: Do you mean `/root/dev` for this, since we still have a dev folder inside root?

Comment (puzzler7) [reply simulation/runSim.sh:5]: /root/dev will also work, and would probably be better - the scene file will always be in the git repo.

File: simulation/runSim.sh (hunk 1 above)
Comment (puzzler7) [reply simulation/runSim.sh:1]: I don't have any opinion - both work. The entire coppelia folder would need to be moved, but that shouldn't be difficult.

File: simulation/runSim.sh (hunk 2 above)
Comment (muthuArivoli) [reply simulation/runSim.sh:5]: then, could you also change that as well?

**Pull request merged**